doc/deployment-on-local-cluster.md

```diff
@@ -0,0 +1,59 @@
+# Deployment on Local Cluster
+
+This project allows you to spin up virtualized Kubernetes cluster, using 
+[kubevirtci](https://github.com/kubevirt/kubevirtci) to deploy a local cluster.
+
+## Ephemeral cluster
+
+```shell
+# Deploy local Kubernetes cluster
+make cluster-up
+
+# Build project, build images, push them to cluster's registry and install them
+make cluster-sync
+
+# SSH to node01 and open interactive shell
+./cluster/cli.sh ssh node01
+
+# Run unit-test
+make test
+
+# Run function test
+make functest
+
+# SSH to node01 and run command
+./cluster/cli.sh ssh node01 echo 'Hello World'
+
+# Communicate with the Kubernetes cluster using kubectl
+./cluster/kubectl.sh
+
+# Destroy the cluster
+make cluster-down
+```
+
+## Check deployment
```

Comment:
Is this really needed? Plus, it is hard to follow what is the purpose of all those snippets below. I would vote to drop it and instead refer to the tutorials as "next steps"

Reply:
I agree. it doesn't really have much value.